gef-ui/api-tests/test-helpers/storage/storage.js

```diff
@@ -0,0 +1,41 @@
+const redis = require('redis');
+const { generateUserSession } = require('../user-session-generator');
+
+const connect = () => {
```

Comment:
There is an issue here that occurs when multiple tests are ran (and very rarely when a single test is run). 

I assume it is an async issue with db writing. Will have a look Monday